scripts/setup_osx.sh

```diff
@@ -23,4 +23,4 @@ echo "Creating conda environment..."
 conda env create -f environment.yml
 conda env update
 
-echo "Conda environment created! Make sure to run \`source activate rllab3\` whenever you open a new terminal and want to run programs under rllab."
+echo "Conda environment created! Make sure to run \`source activate rllab-multiagent\` whenever you open a new terminal and want to run programs under rllab."
```

Comment:
Do we use this? I think we don't, but if so, change `rllab-multiagent` to `flow`.